src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -117,22 +123,29 @@ function ExpensifyCardPage({
                                         domain={domain}
```

Comment:
Looks like this block should have only <DangerCardSection/> or we will have two same blocks on the page

Reply:
sorry, don't follow exactly you have in mind :)

Reply:
thank you, fixed!